requirements.txt

```diff
@@ -0,0 +1,3 @@
+aiohttp~=3.8
+discord.py~=2.0
+async-timeout~=4.0
```

Comment:
What are `aiohttp` and `async-timeout` used for?

Reply:
Whoops, they're requirements for discord.py but they'd be included as its own dependencies. Removed.